src/org/labkey/test/tests/MessagesLongTest.java

```diff
@@ -424,6 +439,37 @@ public void testSteps()
         assertTextPresent("The following new posts were made yesterday");
     }
 
+    @Test
+    public void testDailyDigestMessage()
+    {
+        goToProjectHome();
+        log("Check email preferences");
+        _portalHelper.clickWebpartMenuItem("Messages", true, "Email Preferences");
+        checkCheckbox(Locator.radioButtonByName("notificationType").index(1));
+        clickButton("Update");
+        clickButton("Done");
+
+        SiteNavBar siteNavBar = new SiteNavBar(getDriver());
+        siteNavBar.enterPageAdminMode();
+        log("Customize message board");
+        _portalHelper.clickWebpartMenuItem("Messages", true, "Admin");
+        new AdminPage(getDriver())
+                .includeGroups(true)
+                .save();
+        siteNavBar.exitPageAdminMode();
```

Comment:
Don't bother with all the page admin stuff. You can go straight to the messages admin page.
```
AdminPage.beginAt(this).includeGroups(true).save();
```